addon/-private/formatters/format-relative.ts

```diff
@@ -36,7 +36,7 @@ export default class FormatRelative extends Formatter<RelativeTimeFormatOptions>
     const { format } = formatOptions;
     let unit = formatOptions.unit;
     let opts: RelativeTimeFormatOptions = formatOptions;
-    if (!unit && format && intl.formats.relative && (opts = intl.formats.relative[format])) {
+    if (!unit && format && intl.formats.relative && (opts = intl.formats.relative[format]!)) {
```

Comment:
An alternative here would be to update line 39 to allow `undefined` options

```ts
let opts: RelativeTimeFormatOptions | undefined = formatOptions;
```

Which also matches the current behaviour

Reply:
yeah that's actually better